CHANGELOG.md

```diff
@@ -10,7 +10,9 @@
 
 ## Unreleased
 
+- Remove comments produced by Microsoft Word 2016 (PR #40)
 - Fix dist file containing ES2015 and regex breaking in IE11 (PR #39)
+- Remove more undesirable elements and trim output whitespace (PR #38)
```

Comment:
This should come with PR #38

Reply:
It was easier to build on top of PR #38 for this. Sorry I should have made that clear in the PR.